eng/Dependencies.props

```diff
@@ -15,173 +15,193 @@ and are generated based on the last package release.
-    <LatestPackageReference Include="xunit.extensibility.core" Version="$(XunitExtensibilityCorePackageVersion)" />
-    <LatestPackageReference Include="xunit.extensibility.execution" Version="$(XunitExtensibilityExecutionPackageVersion)" />
-    <LatestPackageReference Include="Microsoft.Data.SqlClient" Version="$(MicrosoftDataSqlClientPackageVersion)" />
+    <LatestPackageReference Include="AngleSharp" />
+    <LatestPackageReference Include="BenchmarkDotNet" />
+    <LatestPackageReference Include="CommandLineParser" />
+    <LatestPackageReference Include="FSharp.Core" />
+    <LatestPackageReference Include="Google.Protobuf" />
+    <LatestPackageReference Include="Grpc.AspNetCore" />
+    <LatestPackageReference Include="Grpc.Auth" />
+    <LatestPackageReference Include="Grpc.Net.Client" />
+    <LatestPackageReference Include="Grpc.Tools" />
+    <LatestPackageReference Include="IdentityServer4" />
+    <LatestPackageReference Include="IdentityServer4.AspNetIdentity" />
+    <LatestPackageReference Include="IdentityServer4.EntityFramework" />
+    <LatestPackageReference Include="IdentityServer4.EntityFramework.Storage" />
+    <LatestPackageReference Include="IdentityServer4.Storage" />
+    <LatestPackageReference Include="Libuv" />
+    <LatestPackageReference Include="MessagePack" />
+    <LatestPackageReference Include="MessagePackAnalyzer" />
+    <LatestPackageReference Include="Mono.Cecil" />
+    <LatestPackageReference Include="Moq" />
+    <LatestPackageReference Include="Newtonsoft.Json.Bson" />
+    <LatestPackageReference Include="NSwag.ApiDescription.Client" />
+    <LatestPackageReference Include="Selenium.Support" />
+    <LatestPackageReference Include="Selenium.WebDriver" />
+    <LatestPackageReference Include="Selenium.WebDriver.ChromeDriver" />
+    <LatestPackageReference Include="Selenium.WebDriver.MicrosoftDriver" />
+    <LatestPackageReference Include="Serilog.Extensions.Logging" />
+    <LatestPackageReference Include="Serilog.Sinks.File" />
+    <LatestPackageReference Include="StackExchange.Redis" />
+    <LatestPackageReference Include="System.Reactive.Linq" />
+    <LatestPackageReference Include="xunit.abstractions" />
+    <LatestPackageReference Include="xunit.analyzers" />
+    <LatestPackageReference Include="xunit.assert" />
+    <LatestPackageReference Include="xunit.extensibility.core" />
+    <LatestPackageReference Include="xunit.extensibility.execution" />
+    <LatestPackageReference Include="Microsoft.Data.SqlClient" />
   </ItemGroup>
 
+  <ItemGroup Label="Dependencies with versions.">
+    <!-- Get name prefixes for version properties. -->
+    <LatestPackageReference Update="@(LatestPackageReference)">
+      <VersionName>$([System.String]::new('%(Identity)').Replace('.',''))</VersionName>
+    </LatestPackageReference>
+
+    <!-- Get versions. -->
+    <LatestPackageReference Update="@(LatestPackageReference)">
+      <Version>$(%(VersionName)PackageVersion)</Version>
```

Comment:
This scares me a _little_ bit - how strong is the guarantee that package version names will always follow this scheme? I believe darc/maestro require it (CC @mmitche), but it seems like we could easily break it for manual version updates like xunit.

Reply:
If we break it, `@(Reference)` items fail loudly.